src/components/AccountsList/useStyles.js

```diff
@@ -2,6 +2,20 @@ import { makeStyles } from "@material-ui/core/styles";
 
 const useStyles = makeStyles(({ typography, breakpoints }) => ({
   root: {},
+  section: {
+    margin: `${typography.pxToRem(20)} 0`,
+    display: "flex",
+    justifyContent: "space-between",
+    padding: 0,
+  },
```

Comment:
Aren't we supposed to add spacing to around `title`? Why are we adding it to `section`?

Reply:
@kilemensi I think it works without any spacing both on the title or section. Removed i
<img width="446" alt="Screenshot 2022-01-28 at 15 51 44" src="https://user-images.githubusercontent.com/3693563/151550342-163be52c-d41f-4d7d-9827-3bf445e68f7e.png">
t 👇🏽